jdtls.ext/com.microsoft.jdtls.ext.core/src/com/microsoft/jdtls/ext/core/PackageCommand.java

```diff
@@ -354,6 +354,25 @@
         return Collections.emptyList();
     }
 
+    private static IPackageFragmentRoot getPackageFragmentRootFromQuery(PackageParams query) {
+        IPackageFragmentRoot packageRoot = (IPackageFragmentRoot) JavaCore.create(query.getHandlerIdentifier());
+        if (packageRoot != null) {
+            return packageRoot;
+        }
+
+        // jar in Referenced Libraries must be constructed from path
```

Comment:
When will ` JavaCore.create(query.getHandlerIdentifier())` return null?

Reply:
e.g. when the input is null